src/tech.js

```diff
@@ -161,7 +161,7 @@ const techs = {
         era: 'civilized',
         reqs: { transport: 1 },
         condition(){
-            return global.tech['farm'] || global.tech['s_lodge'] || (global.tech['hunting'] && global.tech.hunting >= 2) || (global.race['soul_eater'] && !global.race.species === 'wendigo' && global.tech.housing >= 1 && global.tech.currency >= 1) ? true : false;
+            return global.tech['farm'] || global.tech['s_lodge'] || (global.tech['hunting'] && global.tech.hunting >= 2) || (global.race['soul_eater'] && global.race.species !== 'wendigo' && global.tech.housing >= 1 && global.tech.currency >= 1) ? true : false;
```

Comment:
The ternary operator at the end seems unnecessary.
```suggestion
            return global.tech['farm'] || global.tech['s_lodge'] || (global.tech['hunting'] && global.tech.hunting >= 2) || (global.race['soul_eater'] && global.race.species !== 'wendigo' && global.tech.housing >= 1 && global.tech.currency >= 1);
```

Reply:
I'm inclined to agree, though there are many other tech conditions using the ternary operator in the exact same manner. For example, the governor tech condition:
`return global.genes['governor'] && global.civic.govern.type !== 'anarchy' ? true : false;`
Seems odd to only clean up this instance of it.

Reply:
> Seems odd to only clean up this instance of it.

It's the only instance I've checked. I'd be in favor of fixing any and all of them.